### PR TITLE
Fix: Set Planter Manager not update the selfie without Tree Manager Role

### PR DESCRIPTION
**Pull request by tranquanghuy0801**
Resolves issue #57

### CODE REVIEW

**Comment by tranquanghuy0801** [comment]
@nmcharlton  I have also added another PR in Admin API

File: src/components/EditPlanter.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import React, { useState, useEffect } from 'react';
```
Comment (nmcharlton) [inline src/components/EditPlanter.js:1]: There's no need to make any changes to this file.
These changes can be reverted.

File: src/components/EditPlanter.js [hunk 2]
```diff
@@ -91,7 +94,16 @@ const EditPlanter = (props) => {
   }
 
   function handleSelectPlanterImage(img) {
-    handleChange('imageUrl', img);
+    // Planter Manager with Tree Manager role cannot set planter selfie
```
Comment (nmcharlton) [inline src/components/EditPlanter.js:97]: This is not correct.
A planter manager should be able to set this image even if they don't have tree permissions.

File: src/api/planters.js [hunk 3]
```diff
@@ -83,12 +83,13 @@ export default {
       order: 'timeUpdated DESC',
       limit: 100,
       fields: ['planterPhotoUrl'],
-      where: { planterId, planterPhotoUrl: { neq: null } },
     };
 
     const treeQuery = `${
```
Comment (nmcharlton) [inline src/api/planters.js:88]: It doesn't make sense to call this `treeQuery` any more, as it's querying planter selfies.
The client doesn't need to know that the api is using the trees database for this lookup.

**Review by nmcharlton — CHANGES_REQUESTED** [review]

File: src/components/EditPlanter.js (hunk 2 above)
Comment (tranquanghuy0801) [reply src/components/EditPlanter.js:97]: Sorry I wrote wrong comment. It should be "Planter Manager without Tree Manager role cannot set planter selfie". 
So what you mean I should hangle logic when clicking "Save" button instead?

Comment (nmcharlton) [reply src/components/EditPlanter.js:97]: No, I mean that we don’t need to check permissions at all.
The title of the issue is a description of the bug, not the desired functionality.

**Review by nmcharlton — APPROVED** [review]
Great work!

**Pull request merged**